rocketpy/Rocket.py

```diff
@@ -449,6 +460,38 @@ def addMotor(self, motor, position):
         self.evaluateStaticMargin()
         return None
 
+    def addSurfaces(self, surfaces, positions):
+        """Adds one or more aerodynamic surfaces to the rocket. The aerodynamic
+        surface must be an instance of a class that inherits from the
+        AerodynamicSurface
+
+        Parameters
+        ----------
+        surfaces : list, AeroSurfaces, NoseCone, TrapezoidalFins, EllipticalFins, Tail
+            Aerodynamic surface to be added to the rocket. Can be a list of
+            AeroSurfaces if more than one surface is to be added.
+            See AeroSurface class for more information.
+        positions : int, float, list
```

Comment:
See "AeroSurfaces" class instead of "AeroSurface".

Btw, since we have a Motor class instead of Motors class, sholdn't we also name AeroSurface instead of Aerosurfaces?

Sorry for this annoying comment, but it can improve code's consistency

Reply:
Generally, it is good practice keeping class names as a noun in the singular form, so `AeroSurface` would be correct here.

The same should apply to the other classes as well. Nice catch.

Reply:
Should the file `AeroSurfaces.py` be changed to `AeroSurface.py` too? The file current holds several classes

Reply:
@MateusStano , yes the file should be renamed too.
In PEP convention, it should be:
- File: aero_surface.py
- Class: AeroSurface

As we are still not adopting snake_case, it is ok to use AeroSurface.py for now.

But please do not simply change the name of the file, instead use `git rename` so it do not lose the track of commits in the file.

Reply:
Done in https://github.com/RocketPy-Team/RocketPy/pull/355/commits/5b25113544a2e787d9cda07ff113e6fd9c1f8241